Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:12
```

Comment:
Would `node:14` be better? v14 is LTS, while v12 is in maintenance mode.

v14 could also enable `ES2020` as a compile target for tsc: https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping

Reply:
I picked 12 since it's what we were on in sentry. But maybe we could bump both 👀

Reply:
Switched to node 14